scale-down-nodes-vsphere.yml

```diff
@@ -0,0 +1,103 @@
+---
+- hosts: localhost
+
+  vars:
+    workerContent : "{{ lookup('file', '{{ playbook_dir }}/install-dir/worker.64') }}"
+    folder : "/{{ vcenter.datacenter }}/vm/openshift/{{ config.clusterName }}"
+    datastore: "{{ vcenter.datastore }}"
+    govc_file: "/usr/local/bin/govc"
+
+
+  environment:
+    # KUBECONFIG: ./kubeconfig
+    # KUBECONFIG: /root/.kube/config
+    KUBECONFIG: /root/ocp4-vsphere-upi-automation/install-dir/auth/kubeconfig
+
+  tasks:
+    - name: OCPNODES 1.0 | Cordon Nodes
+      shell: oc adm cordon "{{ item.name }}.{{ dns.clusterid }}.{{ dns.domain }}"
+      loop:
+        - { name : "worker3" }
+        - { name : "worker4" }
+        - { name : "worker5" }
+      ignore_errors: yes
+      tags:
+        - "1"
+
+    - name: OCPNODES 1.1 | Drain Nodes
+      shell: oc adm drain "{{ item.name }}.{{ dns.clusterid }}.{{ dns.domain }}" --ignore-daemonsets
```

Comment:
Add `--delete-local-data` here?

Reply:
Maybe not needed when you actually completely delete them anyway...

Reply:
I think it's required if there is state, I will include. Thanks!